packages/mask/src/plugins/Wallet/hooks/useClearRecentTransactions.ts

```diff
@@ -0,0 +1,15 @@
+import { useAccount, useChainId } from '@masknet/web3-shared-evm'
+import { useCallback } from 'react'
+import { WalletRPC } from '../messages'
+
+export function useClearRecentTransactions() {
+    const account = useAccount()
+    const chainId = useChainId()
+
+    const clear = useCallback(async () => {
+        if (!account) return
+        await WalletRPC.clearRecentTransactions(chainId, account)
+    }, [chainId, account])
+
+    return clear
```

Comment:
```suggestion
    return useCallback(async () => {
        if (!account) return
        await WalletRPC.clearRecentTransactions(chainId, account)
    }, [chainId, account])
```

Reply:
I prefer to keep the variable name so that I can easily add `debugger` statement or add breakpoint to debugger, or add log. And the variable name can document itself.

Most minify tools are smart enough to reduce vars at build time.

<img width="811" alt="image" src="https://user-images.githubusercontent.com/1141198/162957351-01a1ecbf-305e-435a-a1c1-8e73db503e99.png">

It's alright to remove  the variable.